### DISCUSSION TITLE
Saturday, October 12, 2024

### DISCUSSION THREAD

**Discussion by wrashi** [GothenburgBitFactory/taskwarrior#3643] [Community Meetings]
Our bi-weekly community meeting happens this Saturday, October 12, 2024.
Reminder that the community meeting changed time from 1200 UTC to 1600 UTC. 

Here is a link where you can check out what that means in your timezone: [World Clock Meeting Times](https://www.timeanddate.com/worldclock/meetingtime.html?month=10&day=12&year=2024&p1=5&p2=3904&p3=3918&p4=3921&iv=0).
The meeting details are below.

Google Meet joining info
Video call link: https://meet.google.com/tns-suqq-iaj
Or dial: ‪(CA) +1 289-323-1423‬ PIN: ‪830 535 232‬#
More phone numbers: https://tel.meet/tns-suqq-iaj?pin=8726596308824

To get a sense of what the meeting is like, you can check out some of latest meetings here:
[TaskWarrior Community Meetings YouTube Playlist](https://www.youtube.com/playlist?list=PLU-esvACzq9XBiGjDCQnMQigKrjlNBq2W).

**Comment by wrashi** [comment]
Looking forward to seeing everyone at our community meeting in 4 hours!

**Comment by wrashi** [comment]
The video from the community meeting is now available 
https://www.youtube.com/watch?v=mm6SX7CTONk

Chad from the [Vit](https://github.com/vit-project/vit) project dropped by and asked terrific questions, so the conversation went a little long. Enjoy!

**Reply by djmitche** [reply]
Much more important that we have good conversations than that they fit in the agenda :)

**Reply by wrashi** [reply]
And this was a great conversation, IMO!
